fission-cli/library/Fission/CLI/App.hs

```diff
@@ -71,7 +71,7 @@ interpret baseCfg cmd = do
       attempt App.Env.read >>= \case
         Right Env {appURL, ipfsIgnored} ->
           run' . local (addAppIgnore ipfsIgnored) $ -- Local because only need to add for this one scenario
-            Handler.publish watch runIO appURL filePath updateDNS updateData
+            Handler.publish open watch runIO appURL filePath updateDNS updateData
```

Comment:
Off topic: one of these days we'll write a `MonadBase` instance for `FissionCLI` and never have to pass these ugly `runIO`s around again.